specs/order_spec.rb

```diff
@@ -78,33 +80,44 @@
   end
 end
 
-# TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Order Wave 2" do
+describe "Order Wave 2" do
   describe "Order.all" do
     it "Returns an array of all orders" do
-      # TODO: Your test code here!
+      result = Grocery::Order.all
+      result.must_equal (CSV.read("/support/orders.csv"))
     end
 
     it "Returns accurate information about the first order" do
-      # TODO: Your test code here!
+      result = Grocery::Order.all
+      result_first = result[0]
+      test_result = CSV.read("/support/orders.csv")
+      test_first = test_result[0]
+      result_first.must_equal test_first
     end
 
     it "Returns accurate information about the last order" do
-      # TODO: Your test code here!
+      result = Grocery::Order.all
+      result_first = result[-1]
+      test_result = CSV.read("/support/orders.csv")
+      test_first = test_result[-1]
+      result_first.must_equal test_first
     end
   end
 
   describe "Order.find" do
     it "Can find the first order from the CSV" do
-      # TODO: Your test code here!
+      result = Grocery::Order.find(1)
+      result.must_equal ([1, {"Slivered Almonds":22.88, "Wholewheat flour":1.93, "Grape Seed Oil":74.9}])
```

Comment:
Test not passing, `find` should return an `Order` and not a 2-element array with a number and a hash.

So I would do
```ruby
result.id.must_equal 1
result.products["Slivered Almonds"].must_equal 22.88
```